actions/templates/list/index.js

```diff
@@ -142,9 +142,6 @@ async function main(params) {
       orderByArray.forEach(item => {
         let field = orderByFields[item[0]].field;
         let dir = item[1];
-        if (orderByFields[item[0]].subfield) {
```

Comment:
Removed because this code is not reachable as there are no fields in `orderByFields` above that have the property `subfield`